management/run.sh

```diff
@@ -2,7 +2,7 @@
 root=$(pwd)
 workdir="$root/src/main/proto"
 javaPackageName='org.thethingsnetwork.management.proto'
-branch='master'
+branch='feature/pprof'
```

Comment:
What should we do in order for this to stay `master`?

Reply:
We have to remove the dependency with the ttn code base. Shall I proceed with that and commit the changes?

Reply:
If you can, please

Reply:
Removed the dependencies with the TTN code